### PR TITLE
Back button for logout

### PR DESCRIPTION
**Pull request by Fang-**
If the back button would log you out, it becomes the eject symbol instead of the back-arrow.

image

### CODE REVIEW

**Comment by vvisigoth** [comment]
I like this and @galenwp was just noting this was a problem. @galenwp any objections to this icon? I personally am neutral to positive.

**Comment by galenwp** [comment]
The icon is fine, but I think it should still say 'Log out' in text. It's
really annoying to log out by accident because you have to enter your
ticket again.

**Comment by Fang-** [comment]
>text _and_ icon

Hmm not sure if I'm a fan...

image

File: src/components/View.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ function View({
   const goBack = useCallback(() => pop(), [pop]);
 
   const showBackButton = size > 1 && !!pop;
+  const backIsLogout = size === 2;
```
Comment (shrugs) [inline src/components/View.js]: should probably be a constant instead of magic variable since it relies on assumptions about our router depth

**Comment by vvisigoth** [comment]
@galenwp @jyng should we decrease the weight of that "Log out"? Speak soon or this is going in, because either way is better than accidentally logging youself out...

**Comment by Fang-** [comment]
image

Note that my changes make the "sidebar" in which the back button lives no longer fixed-size. This means that fluctuations in its size cause content to be pushed off-center for the "log out" case. This isn't ideal.

**Comment by Fang-** [comment]
As discussed with @vvisigoth: last commit needs to be reverted.

**Pull request merged**